### ISSUE TITLE
How can InternVideo be used to extract video features?

### ISSUE THREAD

**Issue by Donnnnnnn** [OpenGVLab/InternVideo#33]
What are the detailed steps and best practices for using the model to effectively extract video features for various video analysis tasks? I would greatly appreciate any guidance or insights. Thank you in advance.

**Comment by shepnerd** [comment]
It depends on your applications. 
1) when you process short videos (lasting for seconds), you can take different sampling methods. If you worked on a typical action recognition, you can extract multiple frames from the video clip and then predict logits from them then average the prediction for the result. If you worked on video retrieval or other applications, you can uniformly sample frames from the video clip. 
2) when you process long videos (more than 1 minute or more), you can partition the video into several clips with or without overlaps, than take the methods in 1) to deal with it.
